aten/src/ATen/native_parse.py

```diff
@@ -71,17 +100,31 @@ def parse_arguments(args, func_decl, func_name, func_return):
         t, name = type_and_name
         default = None
 
+        # Enables Generator? by translating to legacy Generator*. See [temp translations]
+        if t == 'Generator?':
+            t = 'Generator*'
+
         if '=' in name:
             ns = name.split('=', 1)
+            # This enables Tensor? x=None and translates to legacy
+            # "Tensor? x=[]". See [temp translations].
```

Comment:
I'm confused -- "Tensor? x=[]" isn't the legacy -- when I looked in native_functions.yaml now, `Tensor? weight={}` is used.